source/_lovelace/entities.markdown

```diff
@@ -72,20 +72,22 @@ name:
   required: true
   description: Main Label.
   type: string
-icon:
+service:
   required: true
+  description: "Service like `media_player.media_play_pause`"
+  type: string
+icon:
+  required: false
   description: "Icon to display (e.g., `mdi:home`)"
   type: string
+  default: `mdi:remote`
```

Comment:
It can't start with `, leave them away or put it between ""

Reply:
Why can't it start with `?